extensions/amp-base-carousel/0.1/amp-base-carousel.js

```diff
@@ -243,7 +243,8 @@ class AmpCarousel extends AMP.BaseElement {
    *   actionSource: (!ActionSource|undefined),
    * }=} options
    */
-  goToSlide(index, {smoothScroll = false, actionSource} = {}) {
+  goToSlide(index, options = {}) {
+    const {smoothScroll = false, actionSource} = options;
```

Comment:
Could you clarify why this again?

Reply:
The `@param` is named, and it's too difficult to figure out what that is when creating the binding. If it's not named properly, we closure will complain throw.